fitbenchmarking/parsing/parse_nist_data.py

```diff
@@ -303,3 +302,31 @@ def get_startvals_floats(self, startval_str):
             alt_values = [float(startval_str[2])]
 
         return alt_values
+
+    def get_start_x_and_end_x(self, x_data):
+        """
+
+        Get the start and end value of x from the list of x values.
+
+        @param x_data :: list containing x values
+        @return :: the start and end values of the x data
+        """
+        
+        sorted_x_data = sorted(x_data)
+
+        start_x = sorted_x_data[0]
+        end_x = sorted_x_data[-1]
+
+        return start_x, end_x
+
+    def get_data_e(self, data_pattern):
+        """
+
+        :param data_pattern: numpy array of the data points of the problem
```

Comment:
did you mean to write "@param" instead of ":param"?

Reply:
Yes I did mean `@param`

Reply:
@Anders-Markvardsen I have now fixed the comment.